# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: image/FrameAnimator.h
# Commit: 75b04de7e99c
# Full Hash: 75b04de7e99cab7bed5cc8b0f2f6d52ddf174bf3
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-25 09:39:13
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r 91c3acdb2454 -r 75b04de7e99c image/FrameAnimator.h
--- a/image/FrameAnimator.h	Fri May 24 11:32:20 2019 +0000
+++ b/image/FrameAnimator.h	Fri May 24 11:32:54 2019 +0000
@@ -8,13 +8,13 @@
 #define mozilla_image_FrameAnimator_h
 
 #include "mozilla/Maybe.h"
+#include "mozilla/StaticPrefs.h"
 #include "mozilla/TimeStamp.h"
 #include "gfxTypes.h"
 #include "imgFrame.h"
 #include "nsCOMPtr.h"
 #include "nsRect.h"
 #include "SurfaceCache.h"
-#include "gfxPrefs.h"
 
 namespace mozilla {
 namespace image {
@@ -80,7 +80,7 @@
    * Sets the composited frame as valid or invalid.
    */
   void SetCompositedFrameInvalid(bool aInvalid) {
-    MOZ_ASSERT(!aInvalid || gfxPrefs::ImageMemAnimatedDiscardable());
+    MOZ_ASSERT(!aInvalid || StaticPrefs::ImageMemAnimatedDiscardable());
     mCompositedFrameInvalid = aInvalid;
   }
 